src/Pagination/index.jsx

```diff
@@ -292,7 +292,7 @@ class Pagination extends React.Component {
     const { buttonLabels, pageCount } = this.props;
     return (
       <div
-        className="sr-only"
+        className={classNames(styles['sr-only'])}
```

Comment:
Is it necessary to use `classNames` here, instead of assigning `styles['sr-only']` to the `className` prop? E.g.:

```jsx
<div
  className={styles['sr-only']}
  ...
/>

Reply:
Hi @adamstankiewicz . Not at all. I've removed it and verified that it still works.